app/controllers/super_admins_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# super admin controller which cerates new company admin
+class SuperAdminsController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(allowed_user_parameter)
+    @companyadmin=CompanyAdmin.new
```

Comment:
```suggestion
    @companyadmin = CompanyAdmin.new
```
please use rubocop before push